core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -309,4 +309,8 @@ public static void recordAcquireLockFailure(String exceptionClassName) {
 	public static void recordSystemTaskWorkerPollingLimited(String queueName) {
 		counter(classQualifier, "system_task_worker_polling_limited", "queueName", queueName);
 	}
+
+	public static void recordQueuePollSize(String queueType, int val) {
```

Comment:
Please rename method to `recordEventQueuePollSize` to help with maintainability.